services/worker-manager/src/estimator.js

```diff
@@ -0,0 +1,34 @@
+class Estimator {
```

Comment:
I wonder if it would be nice to have a comment here explaining what the thing is supposed to do and why?

Reply:
For sure. I need to add docs all over the place.